util.js

```diff
@@ -37,12 +37,24 @@ async function getAudioLevelMap(videoPath) {
 	for (const frame of intermediateMap) {
 		const clamped = Math.max(Math.min(frame.dBs, average + deviation), average - deviation)
 		const v = Math.abs((clamped - average) / deviation) * 0.5
-		frame.percentMax = clamped > average ? (0.5 + v) : (0.5 - v)
+		frame.percentMax = clamped > average ? 0.5 + v : 0.5 - v
```

Comment:
`0.5 + v : 0.5 - v` is definitely different from `(0.5 + v) : (0.5 - v)`

Reply:
Sorry, I had autoformat on save while working on this and a lot of these changes were made mistakenly, without my knowledge. I didn't notice until later on.